packages/backend/src/controllers/authentication.ts

```diff
@@ -149,15 +149,15 @@ export const redirectOIDCSuccess: RequestHandler = (req, res) => {
             ? decodeURIComponent(req.query.returnTo.toString())
             : undefined;
 
-    if (queryReturn && queryReturn.startsWith(lightdashConfig.siteUrl)) {
+    if (
+        queryReturn &&
+        new URL(queryReturn, lightdashConfig.siteUrl).host ===
```

Comment:
allows `queryReturn` to be path only e.g. `/` or `/path/to/chart` without the host